src/index.js

```diff
@@ -197,9 +203,11 @@ export function createComplexityLimitRule(
           }
 
           if (cost > maxCost) {
-            context.reportError(new GraphQLError(
-              formatErrorMessage(cost), [node],
-            ));
+            context.reportError(
+              createError ?
```

Comment:
It's just a cleaner API. "Modify this error message that someone else created" is not a great look.